wicket-spring-boot-starter-example/src/main/java/com/giffing/wicket/spring/boot/example/web/security/WicketWebSecurityAdapterConfig.java

```diff
@@ -4,51 +4,55 @@
 import org.springframework.context.annotation.Bean;
 import org.springframework.context.annotation.Configuration;
 import org.springframework.security.authentication.AuthenticationManager;
+import org.springframework.security.config.annotation.authentication.configuration.AuthenticationConfiguration;
 import org.springframework.security.config.annotation.web.builders.HttpSecurity;
 import org.springframework.security.config.annotation.web.configuration.EnableWebSecurity;
-import org.springframework.security.config.annotation.web.configuration.WebSecurityConfigurerAdapter;
 import org.springframework.security.core.userdetails.User;
 import org.springframework.security.core.userdetails.UserDetailsService;
 import org.springframework.security.crypto.bcrypt.BCryptPasswordEncoder;
+import org.springframework.security.crypto.password.PasswordEncoder;
 import org.springframework.security.provisioning.InMemoryUserDetailsManager;
 import org.springframework.security.web.firewall.HttpFirewall;
 import org.springframework.security.web.firewall.StrictHttpFirewall;
-
-import com.giffing.wicket.spring.boot.starter.web.servlet.websocket.WicketSessionResolver;
+import org.springframework.security.web.SecurityFilterChain;
 
 /**
  * Default Spring Boot Wicket security getting started configuration. Its only
- * active if there is not other {@link WebSecurityConfigurerAdapter} present.
- * 
+ * active if there is not other {@link SecurityFilterChain} bean is present.
+ *
  * Holds hard coded users which should only be used to get started
  * 
  * @author Marc Giffing
  *
  */
 @Configuration
 @EnableWebSecurity
-public class WicketWebSecurityAdapterConfig extends WebSecurityConfigurerAdapter  {
-	
-	@Override
-	protected void configure(HttpSecurity http) throws Exception {
+public class WicketWebSecurityAdapterConfig {
+
+	@ConditionalOnMissingBean

```

Comment:
formatting

Reply:
There are some problems with the formatting. Maybe its only a eclipse problem but the problem is the usage of tabs and spaces.
![image](https://user-images.githubusercontent.com/3609928/211167467-63b56d1b-7081-4165-9b8b-4ba4688cef7b.png)

I wished I've had used spaces from the beginning...

Reply:
I've restored the tabs layout for the cases above